info.md

```diff
@@ -1,7 +1,11 @@
 ### Bellingham Information
-* Provide Group Information
+* Currently meeting virtually
+* [Become a Member](https://www.owasp.org/index.php/Membership)
+* [Local Chapter Support](https://owasp.org/donate)
+* [Speaker Agreement](https://owasp.org/www-policy/legal/speaker-agreement)
+* [Code of Conduct](https://owasp.org/www-policy/operational/conferences-events.html)
 
 ### Social Links
-* [Meetup](#)
+* [Meetup - coming soon](#)
```

Comment:
Looks like this will be: https://www.meetup.com/owasp-bellingham-chapter/ ?

Reply:
@latf6711 Hi Lisa, let us know if there's any action items for us when it comes to getting the Meetup group going. I've joined as a normal member atm.